main.js

```diff
@@ -415,6 +426,7 @@ app.on('ready', () => {
   logging.initialize().catch((error) => {
     loggingSetupError = error;
   }).then(() => {
+  /* eslint-enable more/no-then */
```

Comment:
What's happening here? Despite the title of the commit ('reducing the scope'), it seems like we're increasing its scope since we're enabling it higher in the file.

Reply:
Yes, we reduce the places where we can use `then`, encouraging new code to use `async` / `await`.